dataland-frontend/src/components/forms/CreateLksgDataset.vue

```diff
@@ -227,7 +226,9 @@ export default defineComponent({
 
       const lksgDataset = (await lkSGDataControllerApi.getCompanyAssociatedLksgData(dataId)).data;
       const dataDateFromDataset = lksgDataset.data?.general?.masterData?.dataDate;
-      this.dataDate = dataDateFromDataset ? new Date(dataDateFromDataset) : this.dataDate;
```

Comment:
Why does it break the tests?

Reply:
It turned out that it does not break the tests. Some tests just accidentally failed after I pushed this change to remote. But actually this change was not the reason for it.
So we could change it to `this.dataDate = dataDateFromDataset ? new Date(dataDateFromDataset) : this.dataDate;` again if you like?

Reply:
(Currently this file `CreateLksgDataset.vue` is unchanged in the PR.)

Reply:
I suspect changes in Lksg in the near future, so maybe let's leave it as it is and change it during Lksg refactoring